### ISSUE TITLE
Test results are messed up under specific conditions.

### ISSUE THREAD

**Issue by just-doit** [utPLSQL/utPLSQL#1060]
To demonstrate the problem I use the following two packages.

```
CREATE OR REPLACE PACKAGE ut_abc
IS
   -- %suite
   -- %suitepath(main.abc)

   -- %test
   PROCEDURE ut_test_01;
END ut_abc;
/

CREATE OR REPLACE PACKAGE BODY ut_abc
IS
   PROCEDURE ut_test_01
   AS
   BEGIN
      ut.expect(TRUE).to_be_true;
   END;
END ut_abc;
/

```
```
CREATE OR REPLACE PACKAGE ut_abc_def
IS
   -- %suite
   -- %suitepath(main.abc_def)

   -- %test
   PROCEDURE ut_test_01;
END ut_abc_def;
/

CREATE OR REPLACE PACKAGE BODY ut_abc_def
IS
   PROCEDURE ut_test_01
   AS
   BEGIN
      ut.expect(TRUE).to_be_true;
   END;
END ut_abc_def;
/

```
The result would correctly look like this:

```
main
  abc_def
    ut_abc_def
      ut_test_01 [,002 sec]
  abc
    ut_abc
      ut_test_01 [,001 sec]
 
Finished in ,006906 seconds
2 tests, 0 failed, 0 errored, 0 disabled, 0 warning(s)

```
But if you set `NLS_SORT` to `GERMAN` you will get:

```
main
  abc_def
    ut_abc
      ut_test_01 [,001 sec]
    ut_abc_def
      ut_test_01 [,001 sec]
  abc
 
Finished in ,006294 seconds
2 tests, 0 failed, 0 errored, 0 disabled, 0 warning(s)

```
BR
Dirk

**Comment by jgebal** [comment]
Thanks for reporing this issue amd getting to the bottom of problem (NLS_SORT)
It's an interesting issue.

Will look into fixing that.

**Comment by jgebal** [comment]
It's resolved in develop branch now.
Will be released as part on v3.1.11.

Thakns again fror finding and reporting this issue.
